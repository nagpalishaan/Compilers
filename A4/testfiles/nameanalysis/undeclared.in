int test(){
    a = 4;
    b = 3;
}