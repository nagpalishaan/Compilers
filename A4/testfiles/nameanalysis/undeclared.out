Error at line 3: expected type does not match. Type: lang.ast.BoolType@5c90e579 != Expected type: lang.ast.IntType@58ea606c
Error at line 3: symbol 'b' is not declared
