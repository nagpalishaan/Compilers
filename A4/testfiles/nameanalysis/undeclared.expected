Error at line 2: symbol 'a' is not declared
Error at line 3: symbol 'b' is not declared
